### PR TITLE
[OPENJDK-2685] Workaround for JDK11 jlinked-image inflating in size

### PR DESCRIPTION
**Pull request by jhuttana**
@Josh-Matsuoka I am unable to select your name from the reviewers list.
Could you also please take a look?

### CODE REVIEW

**Comment by jmtd** [comment]
> @Josh-Matsuoka I am unable to select your name from the reviewers list.

Josh should have an invite from GitHub to the openjdk (container) organisation. Please accept and you can be markd as a reviewer for PRs.

File: modules/jlink/artifacts/opt/jboss/container/java/jlink/mkjreimage.sh [hunk 1]
```diff
@@ -12,4 +12,20 @@ function generate_jre_image() {
     	   	--add-modules "$modules" \
 		--strip-debug --no-header-files --no-man-pages \
 		--compress=2
+
+	# Workaround for JDK11 jlinked-image inflating in size
+	# https://bugzilla.redhat.com/show_bug.cgi?id=1652177
+	# https://bugzilla.redhat.com/attachment.cgi?id=1508295
+	JAVA_MAJOR_VERSION=$(java -version 2>&1 | sed -E -n 's/.* version "([^.-]*).*"/\1/p' | cut -d' ' -f1)
```
Comment (jmtd) [inline modules/jlink/artifacts/opt/jboss/container/java/jlink/mkjreimage.sh]: You should be able to use the variable `JAVA_VERSION` here,  which is defined in the container context by the relevant jdk/jre cekit module.

**Comment by Josh-Matsuoka** [comment]
Looks good to me

**Review by jmtd — APPROVED** [review]
👍🏼

**Pull request merged**